### PR TITLE
[ru] Add translations for Loosely Coupled Architectures

### PR DESCRIPTION
**Pull request by hcz**
### Describe your changes
Russian localisation for:
- [x] Loosely Coupled Architecture

### Related issue number or link (ex: `resolves #issue-number`)
resolves https://github.com/cncf/glossary/issues/3124

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/loosely-coupled-architecture.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Слабо связанная архитектура
+status: Completed
+category: Property
+tags: ["fundamental", "architecture", "property"]
+---
+
+Слабо связанная архитектура (loosely coupled architecture) – это архитектурный стиль,
```
Comment (kirkonru) [inline content/ru/loosely-coupled-architecture.md:8]:
```suggestion
Слабо связанная архитектура (loosely coupled architecture) — это архитектурный стиль,
```

File: content/ru/loosely-coupled-architecture.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+в котором отдельные компоненты приложения независимы друг от друга
+(в противоположность архитектурам с [сильной связанностью компонентов](/tightly-coupled-architecture/)).
+Каждый компонент, иногда называемый микросервисом, создается для выполнения определенной функции с возможностью использования любым количеством других сервисов.
+Этот шаблон, как правило, медленнее внедряется по сравнению с архитектурой с тесно связанными компонентами, но имеет ряд преимуществ, особенно по мере масштабирования приложений.
```
Comment (kirkonru) [inline content/ru/loosely-coupled-architecture.md:12]:
```suggestion
(в противоположность архитектурам с [сильной связанностью компонентов](/ru/tightly-coupled-architecture/)).
Каждый компонент, иногда называемый [микросервисом](/microservices-architecture/), создается под определенную функцию и устроен так, что им могут пользоваться другие сервисы.
При таком подходе реализация, как правило, идет медленнее по сравнению с архитектурой с тесно связанными компонентами, однако у него есть ряд преимуществ, особенно при масштабировании приложений.
```

File: content/ru/loosely-coupled-architecture.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Приложения из слабо связанных компонентов позволяют командам разрабатывать новую функциональность, развертывать и масштабировать их независимо,
+что позволяет организациям быстро, итеративно и независимо улучшать компоненты.
+Таким образом, разработка проходит быстрее, и команды могут быть структурированы вокруг своих компетенций,
+сосредотачиваясь на приложениях в зоне своей отвественности.
```
Comment (kirkonru) [inline content/ru/loosely-coupled-architecture.md:17]:
```suggestion
Приложения из слабо связанных компонентов позволяют командам независимо воплощать новую функциональность, 
проводить развертывания и масштабирования.
Разработка приложений идет быстрее, а команды структурируются в соответствии с их компетенциями, 
фокусируясь на конкретном приложении.
```

**Review by kirkonru — COMMENTED** [review]
This one is hard to translate one-to-one into Russian. I think a slight deviation from the original for the sake of readability is acceptable. What do you think?

**Comment by shurup** [comment]
Thanks for your efforts, @hcz!
With suggestions offered by @kirkonru, LGTM! Let's apply them & merge.

**Comment by kirkonru** [comment]
@hcz Thanks for your contribution! 🙏🏽  The same remark as in your other PR applies here: please use **dev-ru** both as a source to create your own branch and as a target to merge your PR into, since we (RU maintainers) cannot merge into main. IMO, the easiest way to proceed is to close this PR and create a new one with **dev-ru** as a source/target.

**Review by shurup — APPROVED** [review]
LGTM

**Review by kirkonru — APPROVED** [review]

**Pull request merged**